dataprofiler/profilers/profiler_options.py

```diff
@@ -269,6 +269,40 @@ def _validate_helper(self, variable_path: str = "ModeOption") -> list[str]:
         return errors
 
 
+class NumQuantilesOption(BooleanOption):
```

Comment:
I think we call this `QuantilesOption` instead? That way it is extensible to more than just the number of quantiles.

Reply:
Yeah I agree -- more extensible for future development